### PR TITLE
Add mkdir and rename functions to interface

### PR DESCRIPTION
**Pull request by JulianGriggs**
Hi Cerlin,

This purpose of this PR is to add the mkdir and rename function to the interface.
I also updated the return format of the other available API functions (symlink and mtime) in preparation of a future PR.

Let me know if you have any questions!

Julian

### CODE REVIEW

File: Readme.MD [hunk 1]
```diff
@@ -115,7 +127,16 @@ This method is to delete a directory. [CAUTION: Directory has to be empty. (ie,
...
+
+This method is to create a directory.
+
+    /*
+        @param: $akamai_path - Directory path to be added. ex: /cpcode/folder/path
+        @return: Array - ["data" => API response, "code" => API response status code]
+    */
+
+###### 8) Akamai::generateToken($duration, $type);
 [DEPRECATED] Please use Token facade to generate video tokens
```
Comment (talk2cerlin) [inline Readme.MD:141]: Usage sample s missing?

**Review by talk2cerlin — COMMENTED** [review]
I think the usage sample is missing.

File: Readme.MD (hunk 1 above)
Comment (JulianGriggs) [reply Readme.MD:141]: Good point, I'll update

Comment (JulianGriggs) [reply Readme.MD:141]: I have added the usage example.

**Pull request merged**